### ISSUE TITLE
dbx v0.1.0 crashes on Windows platform

### ISSUE THREAD

**Issue by Ceridan** [databrickslabs/dbx#26]
## Problem
`dbx` CLI version 0.1.0 totally crashed on Windows (see traceback below). Previous version (0.0.14) works correctly.

Traceback:
```bash
$ dbx
Traceback (most recent call last):
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\tools\miniconda3\envs\data-platform-pipelines\Scripts\dbx.exe\__main__.py", line 4, in <module>
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\site-packages\dbx\cli.py", line 8, in <module>
    from dbx.commands.datafactory import datafactory
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\site-packages\dbx\commands\datafactory.py", line 8, in <module>
    from azure.identity import DefaultAzureCredential
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\site-packages\azure\identity\__init__.py", line 9, in <module>
    from ._credentials import (
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\site-packages\azure\identity\_credentials\__init__.py", line 5, in <module>
    from .authorization_code import AuthorizationCodeCredential
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\site-packages\azure\identity\_credentials\authorization_code.py", line 8, in <module>
    from .._internal.aad_client import AadClient
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\site-packages\azure\identity\_internal\__init__.py", line 50, in <module>
    from .certificate_credential_base import CertificateCredentialBase
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\site-packages\azure\identity\_internal\certificate_credential_base.py", line 11, in <module>
    from .persistent_cache import load_service_principal_cache
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\site-packages\azure\identity\_internal\persistent_cache.py", line 9, in <module>
    import msal_extensions
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\site-packages\msal_extensions\__init__.py", line 12, in <module>
    from .cache_lock import CrossPlatLock
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\site-packages\msal_extensions\cache_lock.py", line 5, in <module>
    import portalocker
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\site-packages\portalocker\__init__.py", line 4, in <module>
    from . import portalocker
  File "c:\tools\miniconda3\envs\data-platform-pipelines\lib\site-packages\portalocker\portalocker.py", line 9, in <module>
    import win32file
ImportError: DLL load failed while importing win32file: The specified module could not be found.

```

## Your Environment
OS Windows 10, 1909
conda 4.9.2 (using virtual env)
dbx version used: 0.1.0

Actually I created fresh project using [cicd-templates](https://github.com/databrickslabs/cicd-templates)

**Comment by Ceridan** [comment]
Workaround:
```bash
conda install pywin32
```

Fixes the problem for me.

**Comment by renardeinside** [comment]
fixed in #28
